app/javascript/dashboard/store/modules/inboxes.js

```diff
@@ -76,6 +76,13 @@ export const actions = {
       throw new Error(error);
     }
   },
+  updateAutoAssignment: async (nil, { id, ...inboxParams }) => {
+    try {
+      await InboxesAPI.update(id, inboxParams);
```

Comment:
Please update the local state as well. You should be sending `EDIT_INBOXES` mutation